tests/Doctrine/Tests/ORM/Mapping/PHPMappingDriverTest.php

```diff
@@ -33,6 +33,7 @@ protected function _loadDriver()
     public function testinvalidEntityOrMappedSuperClassShouldMentionParentClasses()
     {
         $this->createClassMetadata(DDC889Class::class);
+        $this->addToAssertionCount(1);
```

Comment:
We both know that this is not a valid assertion. Is there some state being created, at least?